src/instructlab/sdg/generate_data.py

```diff
@@ -381,10 +381,12 @@ def generate_data(
 
         if is_knowledge:
             # generate mmlubench data for the current leaf node
+            # note that this needs the original samples dataset as
+            # input, and NOT the newly generated samples dataset
             generate_eval_task_data(
                 mmlu_bench_pipe,
                 leaf_node_path,
-                new_generated_data,
+                ds,
```

Comment:
The auxiliary dataset addition tripped the bug in #202 which caused the generate pipeline to crash, so this fixes that as part of this PR. However, if that issue gets fixed in isolation separately, I'll happily rebase. This gives a chance of CI working on this for now however.

Reply:
I removed this change from my PR since the fixed landed upstream.